modules/applications/examples/ui-builder/development/docker-compose.yml

```diff
@@ -1,10 +1,10 @@
 # Use for running a bonita-ui-builder during development.
 services:
   bonita-ui-builder:
-    image: ${IMAGE_NAME:-pass:a[{bonita-ui-builder-image}]:latest}
+    image: pass:a[{bonita-ui-builder-image}]:latest
```

Comment:
seems weird, from where the image value is taken then ?  did you check this ?

Reply:
The ${IMAGE_NAME} was actually hardcoded.
About the 'pass:a', I actually don't know where it's coming from. 

When you look at the preview, it gives the expected result:

- before: image: ${IMAGE_NAME:-bonitasoft.jfrog.io/docker/bonita-ui-builder:latest}

- after (see preview): image: bonitasoft.jfrog.io/docker/bonita-ui-builder:latest

Reply:
Ha ok !
 I just looked at the source file, and yes this is a antora variable. That's fine then.